content/en/docs/contribute/localization.md

```diff
@@ -73,7 +73,9 @@ For an example of adding a label, see the PR for adding the [Italian language la
 
 ### Find community
 
-Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/). Other localization teams are happy to help you get started and answer any questions you have.
+Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/) and the [SIG Docs Localization Slack channel](https://kubernetes.slack.com/archives/C0191RDKHU1). Other localization teams are happy to help you get started and answer any questions you have.
+
+Please also consider participating in the [SIG Docs Localization Subgroup](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/).  The mission of the SIG Docs localization subgroup is to work across the SIG Docs localization teams to collaborate on defining and documenting the processes for creating localized contribution guides. In addition, the SIG Docs localization subgroup will look for opportunities for the creation and sharing of common tools across localization teams and also serve to identify new requirements to the SIG Docs Leadership team.
```

Comment:
```suggestion
Please also consider participating in the [SIG Docs Localization Subgroup meeting](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/).  The mission of the SIG Docs localization subgroup is to work across the SIG Docs localization teams to collaborate on defining and documenting the processes for creating localized contribution guides. In addition, the SIG Docs localization subgroup will look for opportunities for the creation and sharing of common tools across localization teams and also serve to identify new requirements to the SIG Docs Leadership team.
```

Reply:
Since this mentioned the agenda do you want the folks to consider joining the meeting or what? I’m afraid I’m a bit confused there.

Reply:
Yeah the idea was they may want to participate in the monthly meeting.  How do you think I should clarify this?

Reply:
I wouldn't link to the Google doc from the main website - instead, ask people to join Slack and find it there.

Otherwise, I fear vandalism and crawling.

Reply:
(it may be too late!)

Reply:
Great suggestion @irvifa.  I will send a new update with it now.